testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-excludeIP.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata: 
+  name: whereabouts-excludeip
+spec: 
+  config: '{
+      "cniVersion": "0.3.0",
+      "name": "whereabouts",
+      "type": "macvlan",
+      "mode": "bridge",
+      "ipam": {
+        "type": "whereabouts", 
+        "range": "192.168.10.0/29", 
```

Comment:
@weliang1 Your range has .0 to .7 addresses. Considering 0 and 7 as network and bcast addresses, maximum hosts can be 6. Further we are excluding two hosts .1 and .6. So Pods can take anything from 2 to 5? So 4 pods can be possible via this net-attach-def? But you are expecting 3rd pod to not get IP address which may not be logically correct. Please correct if i am wrong

Reply:
@anuragthehatter 

"range": "192.168.10.0/29", 
        "exclude": [ 
          "192.168.10.0/30",
          "192.168.10.6/32"
        ]

# 192.168.10.0 / 29  HOST RANGE:192.168.10.1 – 192.168.10.6, Broadcast: 192.168.10.7
# 192.168.10.0 / 30  HOST RANGE: 192.168.10.1 – 192.168.10.2, Broadcast:192.168.10.3
# 192.168.10.6 / 32
# Available IP assigned to pod will be: 192.168.10.4 and 192.168.10.5

Reply:
Agree. pardon my calculation..hehe

Reply:
`ipcalc` can help reduce your calculation.:) @anuragthehatter 
`ipcalc 192.168.10.0/29
Network:	192.168.10.0/29
Netmask:	255.255.255.248 = 29
Broadcast:	192.168.10.7

Address space:	Private Use
Address class:	Class C
HostMin:	192.168.10.1
HostMax:	192.168.10.6
Hosts/Net:	6
`